.github/workflows/e2e-tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           restore-keys: ${{inputs.plugin}}
       - name: Install uv
         run: |
-          python -m pip install "uv==0.1.13"
+          python -m pip install "uv~=0.1.24"
```

Comment:
Don't mind making this pin less strict, but the build does seem slower/failing now on the installing dependencies step: https://github.com/kedro-org/kedro-plugins/actions/runs/8438209852/job/23109855477?pr=617 it might just be a glitch though..

Reply:
If it works, don't touch it 😄